docs/reference/gpu_libraries/c++_primitives.md

```diff
@@ -1 +1,40 @@
 # C++ Primitives
+
+ROCm template libraries for algorithms are as follows:
+
+:::::{grid} 1 1 3 3
+:gutter: 1
+
+:::{grid-item-card} rocPRIM
+rocPRIM is an AMD GPU optimized template library of algorithm primitives, like
+transforms, reductions, scans, etc. It also serves as a common back-end for
+similar libraries found inside ROCm.
+
+- [Changelog](https://github.com/ROCmSoftwarePlatform/rocPRIM/blob/develop/CHANGELOG.md)
```

Comment:
What should the order of these be? I know the rocBLAS example had changelog first. Is it best to have the API reference manual first?